fearless/Common/Extension/Storage/Storage+Identifiable.swift

```diff
@@ -6,7 +6,7 @@ extension AccountItem: Identifiable {
 }
 
 extension ConnectionItem: Identifiable {
-    var identifier: String { url.absoluteString }
+    var identifier: String { (url.scheme ?? "") + "://" + (url.host ?? "") }
```

Comment:
what about IPs? I think that's the reason why before it was an absoluteString

Reply:
Didn't get why we need to change that?